### PR TITLE
chore: add volunteer piyushkesarwani

### PR DESCRIPTION
**Pull request by piyushkesarwani**
chore: add volunteer <piyushkesarwani>

### CODE REVIEW

File: data/users/piyushkesarwani.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Piyush Kesarwani",
+  "bio": "Java/C++ Developer | Full Stack Developer ",
+  "image": "https://avatars.githubusercontent.com/u/99383186?s=400&u=f9090471810fcf395c8f91dfa36aa9e291350fc7&v=4",
```
Comment (codewithvoid) [inline data/users/piyushkesarwani.json:4]: use simpler url here 

`https://avatars.githubusercontent.com/u/99383186?v=4`

File: data/users/piyushkesarwani.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Piyush Kesarwani",
+  "bio": "Java/C++ Developer | Full Stack Developer ",
```
Comment (codewithvoid) [inline data/users/piyushkesarwani.json:3]: Pick one title. Preferably `Full Stack Developer`

File: data/users/piyushkesarwani.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  "image": "https://avatars.githubusercontent.com/u/99383186?s=400&u=f9090471810fcf395c8f91dfa36aa9e291350fc7&v=4",
+  "categories": [
+    {
+      "slug": "programming",
+      "topics": ["Java/C++", "JavaScript"]
+    }
+    {
+      "slug": "Software Development",
```
Comment (codewithvoid) [inline data/users/piyushkesarwani.json:11]: use category slugs from this file [Categories](https://github.com/codewithvoid/AskDevs/blob/main/public/categories.json)

File: data/users/piyushkesarwani.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Piyush Kesarwani",
+  "bio": "Java/C++ Developer | Full Stack Developer ",
+  "image": "https://avatars.githubusercontent.com/u/99383186?s=400&u=f9090471810fcf395c8f91dfa36aa9e291350fc7&v=4",
+  "categories": [
+    {
+      "slug": "programming",
+      "topics": ["Java/C++", "JavaScript"]
```
Comment (codewithvoid) [inline data/users/piyushkesarwani.json:8]: mention Java and C++ as separate tags.

Add DSA here only. E.g

```json
"topics": ["Java", "C++", "DSA"]
```

File: data/users/piyushkesarwani.json (hunk 3 above)
Comment (codewithvoid) [reply data/users/piyushkesarwani.json:11]: this category is not required as DSA will be moved

File: data/users/piyushkesarwani.json [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+    }
+    {
+      "slug": "Software Development",
+      "topics": ["Data Structures & Algorithms"]
+    }
+    {
+      "slug": "Web Development",
+      "topics": ["MERN", "JavaScript"]
```
Comment (codewithvoid) [inline data/users/piyushkesarwani.json:16]: - fix slug here too
- remove javascript as its already mentioned in programming

**Comment by codewithvoid** [comment]
Thanks @piyushkesarwani for volunteering. Can you please address the comments.

**Comment by piyushkesarwani** [comment]
Thanks for these valuable suggestions. I have made changes to it.

File: data/users/piyushkesarwani.json (hunk 5 above)
Comment (codewithvoid) [reply data/users/piyushkesarwani.json:12]: Move JavaScript to programming..I would suggest pick 2 of Java, C++ and JavaScript to keep tags max 2-3 per category

**Comment by piyushkesarwani** [comment]
Thanks for the feedback. I've made changes to it.

**Comment by codewithvoid** [comment]
Thanks. Welcome Piyush :)

**Comment by piyushkesarwani** [comment]
Awesome, Thanks!

**Pull request merged**